test/FunctionalTests/Tests/DataOData/Tests/OData.Common.Tests/PublicApi/PublicApi.bsl

```diff
@@ -536,10 +536,10 @@ public enum Microsoft.OData.Edm.EdmOnDeleteAction : int {
 }
 
 public enum Microsoft.OData.Edm.EdmPathTypeKind : int {
-	Annotation = 1

```

Comment:
This seems like a breaking change.

Reply:
`EdmPathTypeKind` is added in my previous PR. https://github.com/OData/odata.net/pull/1051 . It's not in public official release. 

It's a breaking change for my previous PR, not a breaking change for the office release. :)

Thanks for catch this.

#Won't fix.